src/beacon/app/dashboard/analytics/page.tsx

```
@@ -0,0 +1,712 @@
+  Legend,
+  ResponsiveContainer,
+  PieChart,
+  Pie,
+  Cell,
+} from "recharts"
+import {
+  Download,
+  RefreshCw,
+  Activity,
+  Globe,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)